juniper_official/Solutions/Traffic-Blackhole/lookup-interrupts.rule

```diff
@@ -0,0 +1,112 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect Lookup Block CM Errors and Interrupts for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule lookup-interrupts {
+			keys [ fpc-name error-name ];
+			synopsis "Lookup Block CM Errors and Interrupts";
+			description "Lookup Block CM Errors and Interrupts";
+			sensor sensor-interrupts {
+				synopsis "Lookup Block CM Errors and Interrupts";
+				description "Lookup Block CM Errors and Interrupts";
+				open-config {
+					sensor-name /components/component/integrated-circuit/pipeline-counters/errors/lookup-block/lookup-block-error/;
```

Comment:
Needs to be changed to /components/component

Reply:
Changed path to /components/component/